src/etc/inc/certs.inc

```diff
@@ -2072,7 +2072,7 @@ function cert_analyze_lifetime($expiredays) {
 		/* Not expiring soon */
 		$lrclass = 'normal';
 		$expstring = gettext("%d %s until expiration");
-	} elseif ($expiredays > 0) {
```

Comment:
This comparison does not generate an alert on the last day before certificate expiration.